ideascube/utils.py

```diff
@@ -87,3 +88,7 @@ def get_file_sha256(path):
 
 def get_file_size(path):
     return os.stat(path).st_size
+
+def tag_splitter(tag_string):
+    tags = set(t.strip().lower() for t in re.split('[;,]+', tag_string) if t.strip())
```

Comment:
> Can't we lower-case the slug, to avoid duplicates, but keep the visible value the exact way the user typed it?

I'm not sure we can do this. Both `name` and `slug` are "`unique`" field in the `Tag` model.
We could not have a `Tag(name="Bar", slug="bar")` and a `Tag(name="bar", slug="bar")` at the same time.
And the user will asking itself : 
> Why sometime the case isn't changed [no tag already exist with the same slug] and sometime the case is changed, sometime lowering everything, sometime "titleling" it [the tag already exist] ?